spec/blather/client/client_spec.rb

```diff
@@ -1,5 +1,6 @@
 require 'spec_helper'
 require 'blather/client/client'
+require 'active_support/core_ext/string/strip'
```

Comment:
It is used in Line 467: `<<-XML.strip_heredoc`
I didn't try parsing the XML without the strip though, might work anyway, it was just my preferred notation.